pkg/skaffold/parser/config.go

```diff
@@ -129,6 +129,29 @@ func getConfigs(cfgOpts configOpts, opts config.SkaffoldOptions, r *record) (Ska
 		seen[cfgName] = true
 	}
 
+	// validate that manifest files referenced in config exist
+	for _, cfg := range parsed {
+		if cfg.(*latestV1.SkaffoldConfig).Deploy.KubectlDeploy == nil {
+			break
+		}
+		manifests := cfg.(*latestV1.SkaffoldConfig).Deploy.KubectlDeploy.Manifests
+		// TODO(aaron-prindle) check for other manifest types - helm, kpt?
+		for _, pattern := range manifests {
+			if util.IsURL(pattern) {
+				continue
+			}
+			cwd, _ := util.RealWorkDir()
+			expanded, err := filepath.Glob(filepath.Join(cwd, pattern))
+			if err != nil {
+				return nil, err
+			}
+			if len(expanded) == 0 {
+				return nil, sErrors.ConfigHasMissingManifestFileErr(cfgOpts.file, pattern)
+			}
```

Comment:
Looks like the integration tests failed due to 
```
"/home/runner/work/skaffold/skaffold/integration/examples/multi-config-microservices/leeroy-app/skaffold.yaml" referenced file "kubernetes/*" that could not be found. Verify that file "kubernetes/*" referenced in config "/home/runner/work/skaffold/skaffold/integration/examples/multi-config-microservices/leeroy-app/skaffold.yaml" exists and the path and naming are correct.
```
This reminds me, can you also show  the `cwd`  to the user in the error message.
e.g.
```
Verify that file kubernestes/* referenced in config .../multi-config-microservices/leeroy-app/skaffold.yaml 

exists at  /home/runner/work/skaffold/skaffold/integration/examples/multi-config-microservices and the path .....
```

Reply:
Yes, I've added cwd now and there is a bug with multi-config setups:
```
aprindle@aprindle ~/skaffold/examples/multi-config-microservices  [fix-4979_render-should-validate-manifests]$ skaff render --digest-source=none
parsing skaffold config: skaffold config named "/usr/local/google/home/aprindle/skaffold/examples/multi-config-microservices/leeroy-app/skaffold.yaml" referenced file "kubernetes/*" that could not be found. Verify that file "kubernetes/*" referenced in config "/usr/local/google/home/aprindle/skaffold/examples/multi-config-microservices/leeroy-app/skaffold.yaml" exists at "/usr/local/google/home/aprindle/skaffold/examples/multi-config-microservices/kubernetes/*" and the path and naming are correct.
```

Do you know how to get the correct working directory from parser code?  I am not quite sure atm how to do this for multiple config setups, poking around with a debugger now to see what is in `parsed` and why the check
```
		if cfg.(*latestV1.SkaffoldConfig).Deploy.KubectlDeploy == nil {
			break
		}
```
still seems valid for the root skaffold.yaml

Reply:
working now, used `filepath.Dir(cfgOpts.file)` instead of `cwd`  and this resolved this issue.  I'm not quite sure if there are other path related options, etc. to consider.

Reply:
Updated to use `getBase` - https://github.com/GoogleContainerTools/skaffold/pull/5900/files#diff-38a109332f2b81adadb964685493630b15e1a244d53451c897f114be24eb2784R353

as recommended.  Working now for both `-f` flag and default usage

Reply:
I have also added support for absolute paths for the manifest file

Reply:
I strongly feel we shouldn't overload the parser with additional validations. Since the parser is called for all associated commands like `inspect`, `diagnose`, `fix`, which might not need these additional validations up front.

Reply:
In this model,  it needs to be inside the `validation` package and not in the `parser` directly:
https://github.com/gsquared94/skaffold/blob/71ece9d2e37b11a88379c4aa97b5f0e0ec1e2b82/pkg/skaffold/schema/validation/validation.go#L47